src/main/java/core/basesyntax/service/action/BalanceAction.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.service.action;
+
+import static core.basesyntax.model.Storage.storageOfFruits;
+
+public class BalanceAction implements ActionHandler {
+    @Override
+    public void count(String fruit, int amount) {
+        storageOfFruits.put(fruit, amount);
```

Comment:
How will it work with negative amount? The same in other ActionHandler implementations